bin/Bin.re

```diff
@@ -1,3 +1,20 @@
 open Migrate_parsetree;
 
-let _ = Driver.run_as_ppx_rewriter();
+// Source: https://github.com/reasonml-community/graphql_ppx/blob/master/src/bucklescript_bin/Bin.re
+let argv =
+  switch (Sys.argv |> Array.to_list) {
+  | [program, ...rest] =>
+    switch (List.rev(rest)) {
+    | [output_file, input_file, ...args] =>
+      List.concat([
+        [program],
+        List.rev(args),
+        [input_file, "-o", output_file, "--dump-ast"],
+      ])
+      |> Array.of_list
+    | _ => Sys.argv
+    }
+  | _ => Sys.argv
+  };
+
+let () = Driver.run_main(~argv, ());
```

Comment:
I has to use this otherwise there was an error when running on bs side

Reply:
Hmm. I think this is because the BuckleScript compiler is up-to-date with OCaml 4.06: https://bucklescript.github.io/docs/en/upgrade-to-v7#a-quick-look-under-the-hood.

Reply:
This is because you updated `ocaml-migrate-parsetree`. It has nothing to do with BuckleScript. 

https://github.com/ocaml-ppx/ocaml-migrate-parsetree/blob/master/MANUAL.md#new-registration-interface